collator/start-collator.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+docker-compose up -d  #Prometheus and node exporter are
```

Comment:
I think we don't need a script for a single command?

Reply:
Removed